native/ax/TSnative/TSParser.c

```diff
@@ -1975,11 +1975,11 @@ static int ParseData( TS_PARSER* pParser, const unsigned char* pStartData, long
 			bSyncFound = SearchTsHeader( pData, dwSearchBytes, &pHeader );
 
 		if ( bSyncFound ) {
-			used_bytes += (pHeader - pData);
+			used_bytes += (unsigned)(pHeader - pData);

```

Comment:
Please do unsigned int instead of unsigned casts.

Reply:
Done